valhalla/midgard/gridded_data.h

```diff
@@ -394,16 +389,15 @@ template <std::size_t dimensions_t> class GriddedData : public Tiles<PointLL> {
         if (gen_factor > 0.f) {
           Polyline2<PointLL>::Generalize(line, gen_factor, {});
         }
-        // if this ends up as an inner we'll undo this later
```

Comment:
not sure what "we'll undo this later" means but now everything works as expected

Reply:
it was so long ago i cant remember what i was thinking about there :frowning_face:

Reply:
maybe it was a TODO and then I did the todo? but didnt remove the comment? sorry about that